vocabularies/Analytics.xml

```diff
@@ -176,6 +179,129 @@
+/groupby((Country,Region,Segment,Industry),
+         filter($these/aggregate(Amount) gt 0 and
+                $these/aggregate(Currency) ne null))
+/concat(
+  groupby((Country,Region,Segment,Industry))
+    /aggregate($count as LeavesCount),
+  aggregate(Amount,Currency),
+  Analytics.MultiLevelExpand(
+    Levels=[{"DimensionProperties":["Country"],"AdditionalProperties":["CountryName"]},
+            {"DimensionProperties":["Region"],"AdditionalProperties":["RegionName"]},
+            {"DimensionProperties":["Segment","Industry"],"AdditionalProperties":[]}],
+    Aggregation=["Amount","Currency"],
+    SiblingOrder=[{"Property":"Amount","Descending":true}],
+    ShowLevels=2,
+    ExpandEntries=[{"Entry":["US"],"Levels":0},
+                   {"Entry":["DE","BW"],"Levels":1}]
+  )/concat(aggregate($count as ResultEntriesCount),
+           skip(20)/top(10)))
+```
+</String>
+        </Annotation>
+        <Parameter Name="InputSet" Type="Collection(Edm.EntityType)" Nullable="false">
+          <Annotation Term="Core.Description" String="Entity set to be processed" />
+        </Parameter>
+        <Parameter Name="Levels" Type="Collection(Analytics.MultiLevelExpandLevel)" Nullable="false">
+          <Annotation Term="Core.Description" String="Collection of aggregation levels forming a leveled hierarchy" />
+          <Annotation Term="Core.LongDescription">
+            <String>Each element in the collection defines the properties that constitute one level.
+            A property must not referenced by more than one level.
+            The first element in the collection defines the property names of the coarsest level,
+            the following elements define the property names of consecutively finer-grained aggregation levels.
+            The function result is the leveled hierarchy with these levels in preorder,
+            entries on the finest-grained level cannot be expanded further.
+            All referenced properties must be groupable.</String>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="Aggregation" Type="Collection(Edm.String)" Nullable="false">
+          <Annotation Term="Core.Description" String="Properties to aggregate for all result entries on all levels" />
+          <Annotation Term="Core.LongDescription">
+            <String>All properties in this collection must be custom aggregates.</String>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="SiblingOrder" Type="Collection(Analytics.MultiLevelExpandSiblingOrder)" Nullable="false">
+          <Annotation Term="Core.Description" String="Sort specification to apply to all direct descendants of a given entry in the resulting leveled hierarchy" />
+        </Parameter>
+        <Parameter Name="ShowLevels" Type="Edm.Int64" Nullable="false">
+          <Annotation Term="Core.Description" String="Number of levels to be shown in the initial expansion (absent means all levels)" />
```

Comment:
Allowed values range from 0 to length of "Levels", I assume? Should we define this contract more explicit? Or would it be OK to provide even larger values?